copylot/hardware/lasers/vortran/vortran.py

```diff
@@ -7,21 +7,19 @@
 refer to the manuals in https://www.vortranlaser.com/
 
 """
-from copylot.hardware.lasers.abstract_laser import AbstractLaser
+import logging
+import time
 import serial
 from serial.tools import list_ports
-import time
-import logging
+
+from copylot.hardware.lasers.abstract_laser import AbstractLaser
 
 # Setting up the Logger
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 handler = logging.StreamHandler()
 handler.setLevel(logging.DEBUG)
-formatter = logging.Formatter(
-    '%(asctime)s - %(name)s -\
-                                 %(levelname)s - %(message)s'
-)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Comment:
I believe this logger is deprecated, can we update the branch with latest `main`?